api/src/infrastructure/docker.rs

```diff
@@ -361,8 +384,20 @@ impl DockerInfrastructure {
             strategy.container_type(),
         );
 
-        let options =
-            DockerInfrastructure::create_container_options(app_name, strategy, container_config);
+        let host_config_binds = DockerInfrastructure::create_host_config_binds(
+            app_name,
+            image_info,
+            existing_volumes,
+            strategy,
+        )
+        .await?;
+
+        let options = DockerInfrastructure::create_container_options(
```

Comment:
I'm missing the part of deleting the volumes when the app will be shutdown.

Reply:
As discussed, delete_volume_mount is part of stop_services_impl and is now triggered with self.delete_volume_mount(app_name)